package.json

```diff
@@ -97,5 +98,23 @@
     "json-logic-js": "^2.0.2",
     "lodash": "^4.17.21",
     "moment": "^2.29.4"
+  },
+  "nyc": {
+    "check-coverage": true,
+    "statements": 62,
+    "branches": 52,
+    "functions": 57,
+    "lines": 60,
+    "include": [
+      "src/**/*.ts",
+      "src/**/*.js"
+    ],
+    "exclude": [
+      "src/**/*.test.ts",
+      "src/**/__tests__/",
+      "src/experimental/**/*.ts",
+      "src/types/**/*.ts"
+    ],
+    "all": true
   }
```

Comment:
code coverage configuration.  Some of the exclusions should be reviewed.  the coverage thresholds are based on current value based on the entire project.